### DISCUSSION TITLE
Some blogs don't/won't load?

### DISCUSSION THREAD

**Discussion by Someonemustnothavethis** [TumblThreeApp/TumblThree#270] [Q&A (Question & Answer)] [answered]
I've been exploring this for a while and have two good examples.
https://kirstysdreamcloset.tumblr.com/  This is a perfectly innocuous blog full of nice clothes.  I use T3 (TumbleThree) every month to catch up with the posts.  It also shows up in Tumbex's archive view,
https://www.tumbex.com/kirstysdreamcloset.tumblr/posts

Here is another perfectly innocent blog that DOESN'T load into T3,
https://www.tumblr.com/blog/view/sexysilkysatin
Notice that while I'm following it, just like the first blog example, the URL shows 'view'.  I can't make it show up the same as the other blogs I follow.
Further, this second blog also doesn't show up in Tumbex.  It shows up if I search for it,
https://www.tumbex.com/search?q=sexysilkysatin
But Tumbex can't read it's archive view,
https://www.tumbex.com/sexysilkysatin.tumblr/posts

I've noticed that this happens mostly with _some_ adult blogs but it's almost random.  The blog I've mentioned as NOT an adult one and Tumblr doesn't make me click through all the warnings it usually does for those kinda blogs.  I'm guessing that this is some random thing that Tumblrs doing.  Has anyone else noticed this or is it just me?
Yes, T3 is logged in, and I follow both blogs.

Is there a work around for this?
Thanks a bunch

**Comment by thomas694** [comment]
Normally, I add blogs by using "xyz.tumblr.com", especially blogs that are redirected in the browser.

Both blogs can be added and downloaded without a problem here. The first one is a normal and the second one is a hidden blog (login required). So at least the second one needs crawler type SVC (normally chosen automatically when added).

You can edit the files kirstysdreamcloset.tumblr resp. sexysilkysatin.tmblrpriv and check/edit (be sure your editor handles them correctly or make a backup before) the line with the URL:
```
  "Url": "https:\/\/kirstysdreamcloset.tumblr.com\/",
  "Url": "https:\/\/sexysilkysatin.tumblr.com\/",
```
if they contain any other style/pattern.

**Comment by Someonemustnothavethis** [comment]
Thanks for the quick reply Thomas, but I'm not by any means a programmer and most of what you typed after the second line is meaningless to me.  Can you give me the "Guy with a wrench' version?
Thanks!

**Comment by thomas694** [comment]
Maybe I've read over it. Have you already added the 2nd blog to your TumblThree? If not, you can add it by using the standard pattern "blogname.tumblr.com".
If yes, I try to phrase it better...

I did a quick test with your two blogs and added them for testing. In the column "Type" of the blog list, one is shown as "Tumblr" and the 2nd one as "Tumblr Hidden". So the 2nd one requires that in the blog details (on the right) the option "Crawler" is set to "Tumblr SVC" (normally done automatically by the app) and the user is logged in (settings dialog) in order to be able to download the blog.

In both cases, the URL shown in the Url column of the blog list shows as "https://blogname.tumblr.com/" (no "view" or anything else in the URL). If in your case there is something else in the URL, you can fix it by either editing the blog file, which is inside the subfolder "Index" of your blogs "download location", or by removing and adding again the blog (adjusting blog settings again) and moving the already downloaded files from the old download location to the blog's new download location.
For the first option you open the general download location in Windows File Explorer, go to the subfolder "Index", search the corresponding blog file (either named blogname.tumblr or blogname.tmblrpriv), open it in a text editor (e.g. notepad) and edit the line which contains "URL" to the format I've shown above.
Then the download should work without a problem.

**Comment by Someonemustnothavethis** [comment, marked as answer]
Aha!...got it finally.   I hadn't had the URL or Type columns showing and when I enabled them I found that most of my followed blogs are hidden (I guess that says a lot about what I'm downloading on Tumblr).  I followed your instructions and it's now working.   I'll keep the directions handy in case I run into this again although I'm really, truly, honestly, desperately trying to cut back on my Tumblr obsession.
Honestly.
Thanks a bunch, you can close this now.
